app/controllers/activities/uploads_controller.rb

```diff
@@ -33,9 +33,9 @@ def update
     authorize report, :upload?
 
     @report_presenter = ReportPresenter.new(report)
-    upload = CsvFileUpload.new(params[:report], :activity_csv)
-    @success = false
     @type = params[:type].to_sym
+    upload = CsvFileUpload.new(params[:report], :"activity_csv_#{@type}")
```

Comment:
You can symbolise a string like this? How did I not know that!

Reply:
Yep. I don't know if I prefer it to `to_sym`, but sometimes I feel like using it when interpolating